### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by clintonjosephs**
Hello 😄 ,

I was able to work on the to-do list's interactive features to achieve the following:

- Create new module containing methods related to the status updates (completed: true / false) ✅ 
- Updated items object's value for completed key upon user actions.✅ 
- Implement a function for the "Clear all completed" button (using filter() method).✅ 
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.✅ 

Thank you for reviewing my project! 💯

### CODE REVIEW

File: src/Completed.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    storageManager.storeData(toDoList);
+    Methods.uIRefreshInstance(listContainer);
+  }
+
+  static clearCompleted(listContainer) {
+    let toDoList = storageManager.getData();
+    toDoList = toDoList.filter((todo) => todo.completed !== true);
+    storageManager.storeData(toDoList);
+    Methods.uIRefreshInstance(listContainer);
+  }
```
Comment (dasileker) [inline src/Completed.js]: - [x] Please use  Best practice ES6 by using the Arrow functions in all your files.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @clintonjosephs,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Completed.js (hunk 1 above)
Comment (clintonjosephs) [reply src/Completed.js]: Thank you for your review😊

**Review by brytebee — APPROVED** [review]
Hi @clintonjosephs👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**